packages/by-name/cli-release/package.nix

```diff
@@ -12,6 +12,7 @@
     install -D ${lib.getExe genpolicy} cli/cmd/assets/genpolicy
     install -D ${genpolicy.settings}/genpolicy-settings.json cli/cmd/assets/genpolicy-settings.json
     install -D ${genpolicy.rules}/genpolicy-rules.rego cli/cmd/assets/genpolicy-rules.rego
+    echo $"${builtins.readFile ../../../cli/cmd/assets/image-replacements.txt}" > cli/cmd/assets/image-replacements.txt
```

Comment:
Are the nix changes necessary? The file should be right there to begin with...

Reply:
The file is only there locally and not inside the nix build, as I understand it. That's why I have to `touch` the file for dev builds, resulting in an empty image replacement file. Dev builds will not fail, referring to your other comment, because the file is then completely empty.